core/admin/mailu/models.py

```diff
@@ -255,20 +255,23 @@ def dns_autoconfig(self):
         """ return list of auto configuration records (RFC6186) """
         hostname = app.config['HOSTNAME']
         protocols = [
-            ('submission', 587),
-            ('imap', 143),
-            ('pop3', 110),
+            ('imap', 143, 20),
+            ('pop3', 110, 20),
+            ('submission', 587, 20),
         ]
         if app.config['TLS_FLAVOR'] != 'notls':
             protocols.extend([
-                ('imaps', 993),
-                ('pop3s', 995),
+                ('autodiscover', 443, 10),
+                ('submissions', 465, 20, 10),
+                ('imaps', 993, 10),
+                ('pop3s', 995, 10),
             ])
+
         return list([
-            f'_{proto}._tcp.{self.name}. 600 IN SRV 1 1 {port} {hostname}.'
-            for proto, port
+            f'_{proto}._tcp.{self.name}. 600 IN SRV {prio} 1 {port} {hostname}.'
+            for proto, port, prio
```

Comment:
This leads to an error for me when opening the domain details page. I haven't looked at the cause yet.
```
ValueError: too many values to unpack (expected 3)
```

Stack trace:
```
admin_1      | [2022-03-14 07:50:24,334] ERROR in app: Exception on /admin/domain/details/test.berichthier.nl [GET]
admin_1      | Traceback (most recent call last):
admin_1      |   File "/usr/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
admin_1      |     response = self.full_dispatch_request()
admin_1      |   File "/usr/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
admin_1      |     rv = self.handle_user_exception(e)
admin_1      |   File "/usr/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
admin_1      |     rv = self.dispatch_request()
admin_1      |   File "/usr/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
admin_1      |     return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
admin_1      |   File "/usr/lib/python3.9/site-packages/flask_login/utils.py", line 272, in decorated_view
admin_1      |     return func(*args, **kwargs)
admin_1      |   File "/app/mailu/ui/access.py", line 28, in wrapper
admin_1      |     return callback(function, args, kwargs, dargs, dkwargs)
admin_1      |   File "/app/mailu/ui/access.py", line 19, in callback
admin_1      |     return function(*args, **kwargs)
admin_1      |   File "/app/mailu/ui/views/domains.py", line 67, in domain_details
admin_1      |     return flask.render_template('domain/details.html', domain=domain)
admin_1      |   File "/usr/lib/python3.9/site-packages/flask/templating.py", line 147, in render_template
admin_1      |     return _render(
admin_1      |   File "/usr/lib/python3.9/site-packages/flask/templating.py", line 128, in _render
admin_1      |     rv = template.render(context)
admin_1      |   File "/usr/lib/python3.9/site-packages/jinja2/environment.py", line 1304, in render
admin_1      |     self.environment.handle_exception()
admin_1      |   File "/usr/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
admin_1      |     raise rewrite_traceback_stack(source=source)
admin_1      |   File "/app/mailu/ui/templates/domain/details.html", line 1, in top-level template code
admin_1      |     {%- extends "base.html" %}
admin_1      |   File "/app/mailu/ui/templates/base.html", line 70, in top-level template code
admin_1      |     {%- block content %}{%- endblock %}
admin_1      |   File "/app/mailu/ui/templates/domain/details.html", line 24, in block 'content'
admin_1      |     {%- call macros.table(datatable=False) %}
admin_1      |   File "/usr/lib/python3.9/site-packages/jinja2/runtime.py", line 828, in _invoke
admin_1      |     rv = self._func(*arguments)
admin_1      |   File "/app/mailu/ui/templates/macros.html", line 102, in template
admin_1      |     {{- caller() }}
admin_1      |   File "/usr/lib/python3.9/site-packages/jinja2/runtime.py", line 828, in _invoke
admin_1      |     rv = self._func(*arguments)
admin_1      |   File "/app/mailu/ui/templates/domain/details.html", line 65, in template
admin_1      |     {%- for line in domain.dns_autoconfig %}
admin_1      |   File "/usr/lib/python3.9/site-packages/jinja2/environment.py", line 474, in getattr
admin_1      |     return getattr(obj, attribute)
admin_1      |   File "/usr/lib/python3.9/site-packages/werkzeug/utils.py", line 97, in __get__
admin_1      |     value = self.fget(obj)  # type: ignore
admin_1      |   File "/app/mailu/models.py", line 270, in dns_autoconfig
admin_1      |     return list([
admin_1      |   File "/app/mailu/models.py", line 272, in <listcomp>
admin_1      |     for proto, port, prio
admin_1      | ValueError: too many values to unpack (expected 3)
```

Reply:
Fixed in 0e9a1e6a I believe